StoryCardMaker.py

```diff
@@ -74,12 +73,19 @@ def func_characternumber():																		######
 	return y																					# and send back the number of characters	
 
 
+def main():
+	# checks to see if sessions directory exists in current directory, if not -- create one
+	# uses os instead of pathlib for python 2 compatibility
+	try: 
+		os.makedirs('./sessions')
+	except OSError:
+		if not os.path.isdir('./sessions'):
+			raise
 
-						######### The Program Begins			
-func_disptext('opening')	#
-y = func_characternumber()		#
-func_makethelist()			#
```

Comment:
do we need to send "y" value to func_makethelist() ?

Reply:
Ha -- whoops, the whole thing was a little quick and dirty, I'll fix it.